src/app/qgisapp.cpp

```diff
@@ -13409,3 +13419,51 @@ QgsFeature QgisApp::duplicateFeatureDigitized( QgsMapLayer *mlayer, const QgsFea
   return QgsFeature();
 }
 
+
+void QgisApp::populateProjectStorageMenu( QMenu *menu, bool saving )
+{
+  menu->clear();
+  for ( QgsProjectStorage *storage : QgsApplication::projectStorageRegistry()->projectStorages() )
+  {
+    QString name = storage->visibleName();
+    if ( name.isEmpty() )
+      continue;
+    QAction *action = menu->addAction( name );
+    if ( saving )
+    {
+      connect( action, &QAction::triggered, [this, storage]
+      {
+        QString uri = storage->showSaveGui();
+        if ( !uri.isEmpty() )
+        {
+          QgsProject::instance()->setFileName( uri );
+          if ( QgsProject::instance()->write() )
+          {
+            setTitleBarText_( *this ); // update title bar
+            mStatusBar->showMessage( tr( "Saved project to: %1" ).arg( uri ), 5000 );
```

Comment:
Won't this get a bit annoying? I save every 10 seconds sometimes :p

Reply:
We have the same kind of code in QgisApp::fileSave() and QgisApp::fileSaveAs() so I just brought it here as well for consistency... I am happy to remove it if you prefer - but this is just status bar text update - not a popup in the message bar

Reply:
Ah - I read that wrong and thought it was message bar. In that case no issue here!